scripts/newinstall.sh

```diff
@@ -282,11 +306,18 @@ set -e
 ##########	Download optional component (python, git, ...)
 
 if true; then
-	if [[ $WITH_MINICONDA2 = 1 ]]; then
-		echo "Installing Miniconda2 Python Distribution ... "
-		$cmd eups distrib install --repository="$EUPS_PKGROOT" miniconda2 "$MINICONDA2_VERSION"
-		$cmd setup miniconda2
-		CMD_SETUP_MINICONDA2='setup miniconda2'
+	if [[ $WITH_MINICONDA == 1 ]]; then
+		if [[ $USE_PYTHON2 == false ]]; then
+			PYVER_PREFIX=3
+			MINICONDA_VERSION=${MINICONDA3_VERSION}
+		else
+			PYVER_PREFIX=2
+			MINICONDA_VERSION=${MINICONDA2_VERSION}
+		fi
+		echo "Installing Miniconda${PYVER_PREFIX} Python Distribution ... "
+		$cmd eups distrib install --repository="$EUPS_PKGROOT" "miniconda${PYVER_PREFIX}" "$MINICONDA_VERSION"
+		$cmd setup "miniconda${PYVER_PREFIX}"
+		CMD_SETUP_MINICONDA='setup miniconda${PYVER_PREFIX}'
```

Comment:
Looks like the here document below uses "CMD_SETUP_MINICONDA2"; that'll need changing.